weblayer/public/java/org/chromium/weblayer/BrowserFragment.java

```diff
@@ -117,12 +117,12 @@ void onCreate(Bundle savedInstanceState, @Nullable TabListCallback tabListCallba
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
```

Comment:
hmm

Reply:
Our unit tests were failing because it didn't register that the `super` method was being called in another function. Mainly due to a linting error

Reply:
Oh, I see. This isn't changing anything functional so LGTM